### ISSUE TITLE
Zombies can't do anything about Felinids in bags

### ISSUE THREAD

**Issue by DangerRevolution** [DeltaV-Station/Delta-v#1108]
## Description

title, this is bad because it can lead to round stalling and it's just generally unfun
**Reproduction**

be zombie
come across felinid in bag
you can't do anything

**Comment by Bonktrauma** [comment]
I agree this is an issue, only thing they can do is save the area. If they have a hardsuit they can't so anything

**Comment by Axionyxx** [comment]
Can't you break the bag? I'm not sure if they're destructible or not... A system to transfer damage from the bag to the felinid would also work

**Comment by Bonktrauma** [comment]
> Can't you break the bag? I'm not sure if they're destructible or not... A system to transfer damage from the bag to the felinid would also work

Bags are indestructible

**Comment by Axionyxx** [comment]
> > Can't you break the bag? I'm not sure if they're destructible or not... A system to transfer damage from the bag to the felinid would also work
> 
> Bags are indestructible

Damn, where can I get one?

**Comment by Adrian16199** [comment]
to be fully frank, if its an issue that zombies cant bite the bag, there is a possibility of doin somethin similar for items inside to inherit any damage done on the bag maybe.
Code that explosives deal damage to stuff inside a bag is a thing, one could modify it to include everything.

**Comment by SolStar2** [comment]
I think this can be fixed by having a do-after action to empty the contents on the ground. It would also work for sec.

**Comment by SolStar2** [comment]
Similar to plant bag or ore bag

**Comment by Vapetastic-Gaming** [comment]
Depreciated Issue, has been addressed by direction, will possibly be grandfathered in, or fixed in the future
